modules/media/jsx/columnFormatter.js

```diff
@@ -18,12 +18,19 @@ function formatColumn(column, cell, rowData, rowHeaders) {
     row[header] = rowData[index];
   }, this);
 
+  // create array of classes to be added to td tag
+  var classes = [];
+  if (row['Hide File'] === '1') {
+    classes.push("bg-danger");
+  }
+  // convert array to string, with blank space separator
+  classes = classes.join(" ");
```

Comment:
this seems overly complicated, I don't see any code paths where this would ever contain more than 1 class. It's either empty, or "bg-danger", so why bother with the array/join?